website/docs/feature-flag-tutorials/rust/implementing-feature-flags-rust.md

```diff
@@ -200,26 +197,24 @@ cargo add tokio --features full
 cargo add unleash-api-client --features reqwest
 ```
 
-There are a few dependencies, and here's why: We need an HTTP client to make the request, and `serde` to deserialize the Unleash response. Our SDK is constantly polling Unleash to retrieve the value of feature flags, and caches that in memory. 
+There are a few dependencies, and here's why: We need an HTTP client to make the request, and `serde` to deserialize the Unleash response. Our SDK is constantly polling Unleash to retrieve the value of feature flags, performs its own evaluation, and caches that in memory.
```

Comment:
Subtle thing here but probably mostly just for your understanding - the flags are cached, the evaluation is done on the fly and the results of that are never cached